cookie.go

```diff
@@ -14,7 +31,7 @@ func (c *cookie) get(ctx *fasthttp.RequestCtx, name string) []byte {
 	return ctx.Request.Header.Cookie(name)
 }
 
-func (c *cookie) set(ctx *fasthttp.RequestCtx, name string, value []byte, domain string, expiration time.Duration, secure bool) {
+func (c *cookie) set(ctx *fasthttp.RequestCtx, name string, value []byte, domain string, expiration time.Duration, secure bool, sameSite string) {
```

Comment:
Could you change the `sameSite` type from `string` to `CookieSameSite` type please?